graphics/fonts/winfont.cpp

```diff
@@ -310,4 +321,17 @@ void WinFont::drawChar(Surface *dst, uint32 chr, int x, int y, uint32 color) con
 	}
 }
 
+int WinFont::getStyle() {
+	int style = kFontStyleRegular;
+
+	if (_weight >= 700)
```

Comment:
Please add here reference to Wine source, so we know where it comes from. Please, link to GitHub including line number.